x/ccv/provider/keeper/relay.go

```diff
@@ -256,6 +259,52 @@ func (k Keeper) QueueVSCPackets(ctx sdk.Context) {
 	k.IncrementValidatorSetUpdateId(ctx)
 }
 
+func (k Keeper) ProviderValidatorUpdates(ctx sdk.Context) []abci.ValidatorUpdate {
+	// get the bonded validators from the staking module
+	bondedValidators := k.stakingKeeper.GetBondedValidatorsByPower(ctx)
+
+	// get the last validator set sent to consensus
+	currentValidators := k.GetLastProviderConsensusValSet(ctx)
+
+	nextValidators := []types.ConsumerValidator{}
+	maxValidators := k.GetMaxProviderConsensusValidators(ctx)
+	// avoid out of range errors by bounding the max validators to the number of bonded validators
+	if maxValidators > int64(len(bondedValidators)) {
+		maxValidators = int64(len(bondedValidators))
+	}
+	for _, val := range bondedValidators[:maxValidators] {
+		// create the validator from the staking validator
+		consAddr, err := val.GetConsAddr()
```

Comment:
You could potentially reuse [`CreateConsumerValidator`](https://github.com/cosmos/interchain-security/blob/98bd90f63b77d2c5571af11f14786064d77bf555/x/ccv/provider/keeper/validator_set_update.go#L137) with an empty `chainID`.